app/lib/publisher/backend.dart

```diff
@@ -93,6 +96,40 @@ class PublisherBackend {
     return _asPublisherInfo(p);
   }
 
+  /// Invites a user to become a publisher admin.
+  Future<account_api.InviteStatus> invitePublisherMember(
+      String publisherId, api.InviteMemberRequest invite) async {
+    return await _withPublisherAdmin(publisherId, (p) async {
+      InvalidInputException.check(
+          isValidEmail(invite.email), 'Invalid e-mail: `${invite.email}`');
+      final user = await accountBackend.lookupOrCreateUserByEmail(invite.email);
+
+      final userId = user.userId;
+      final key = p.key.append(PublisherMember, id: userId);
+      final pm = (await _db.lookup<PublisherMember>([key])).single;
+      if (pm != null) {
+        InvalidInputException.check(pm.isPending, 'User is already a member.');
```

Comment:
If already a member, then isn't that just fine? If we want to return some error it's a conflict.

IMO, it would be okay to do this for the case where the invite has been accepted, otherwise, keep idempotency in mind.

Reply:
This check will be removed once we remove `isPending`.